ui/js/controllers/jobs.js

```diff
@@ -67,7 +67,11 @@ treeherderApp.controller('JobsCtrl', [
             }
         };
 
-        if(ThResultSetStore.isNotLoaded($scope.repoName)){
+        if ($location.search().revision === 'undefined') {
+            thNotify.send("Revision Undefined", 'danger');
```

Comment:
I might say something like "Invalid value for revision parameter." here.  I think that might clue the user in better.